### ISSUE TITLE
Criterion problem

### ISSUE THREAD

**Issue by chihhuiho** [corenel/pytorch-adda#11]
Thanks for sharing the code. I have a question about the line 25 in adapt.py (criterion = nn.CrossEntropyLoss() ). Since nn.LogSoftmax() is added in discriminator.py (line 21). Is it correct to use CrossEntropyLoss as criterion ? Or maybe I miss something? Thanks in advance.

**Comment by jinfengr** [comment]
The same question here, I also tried NLLLoss()

**Comment by sungyubkim** [comment]
I think GAN loss of this implementation is not correct.

**Comment by TuGang-Git** [comment]
> The same question here, I also tried NLLLoss()

How was the result?
